src/test/regress/sql/seclabel.sql

```diff
@@ -82,6 +91,21 @@ SELECT 1 FROM citus_add_node('localhost', :worker_2_port);
 SELECT node_type, result FROM get_citus_tests_label_provider_labels('user1') ORDER BY node_type;
 SELECT node_type, result FROM get_citus_tests_label_provider_labels('"user 2"') ORDER BY node_type;
 
+-- disable the GUC and check that the command is not propagated
+SET citus.enable_alter_role_propagation TO off;
+SECURITY LABEL ON ROLE user1 IS 'citus_unclassified';
+SELECT node_type, result FROM get_citus_tests_label_provider_labels('user1') ORDER BY node_type;
+RESET citus.enable_alter_role_propagation;
+RESET citus.log_remote_commands;
```

Comment:
we don't have to reset them given that we're anyway switching the session at Line 101